iceoryx_examples/ice_access_control/README.md

```diff
@@ -93,7 +95,7 @@ roudiConfig.m_sharedMemorySegments.push_back({"infotainment", "infotainment", me
 ```
 
 The `roudiConfig` is composed of a memory pool config called `mepooConfig`. When the segement is created, one needs to
```

Comment:
```suggestion
The `roudiConfig` is composed of a memory pool config called `mepooConfig`. When the segment is created, one needs to
```

Reply:
Oh boy, even the cpp name was wrong :smile: Is this a bob?